test/built-ins/RegExp/prototype/Symbol.replace/poisoned-stdlib.js

```diff
@@ -17,6 +17,17 @@ assert(delete String.prototype.indexOf);
 assert(delete String.prototype.slice);
 assert(delete String.prototype.substring);
 
+for (let i = 0; i < 5; ++i) {
+    Object.defineProperty(Array.prototype, i, {
+        get: function() {
+            throw new Test262Error(i + " getter should be unreachable.");
+        },
+        set: function(_value) {
+            throw new Test262Error(i + " setter should be unreachable.");
```

Comment:
While JSC uses private copy of `Array.prototype.push` that is unaffected by `delete` at L11, it still performs the [`push()`](https://tc39.es/ecma262/#sec-array.prototype.push) which does `[[Set]]` at step **5.a**.